flybywiresim/a32nx
# refactor(ecam): start moving individual pages to ReactJS

Closes #2026
Closes #2518

## Summary of Changes

This PR allows ReactJS pages part of the ECAM rewrite to be individually moved over and integrated into the Asobo ECAM.

The first page to be moved (as part of this same PR) is the DOOR/OXY SD page.

When all pages are moved, we will replace the entire SD with a ReactJS instrument.

Credit for the rewritten DOOR/OXY page goes to @RichardPilbery.

## Screenshots (if necessary)

![image](https://user-images.githubusercontent.com/4503241/107134133-458e6680-68bd-11eb-9277-e2127917820d.png)o find references for what you are working on. Exceptions will probably be made for IRL A320 pilots and engineers. -->

Discord username (if different from GitHub): someperson#4953

## Testing instructions


* Make sure that the DOOR/OXY page functions correctly
* Make sure that all other ECAM SD pages function correctly
* Make sure that no performance drops are seen
* Make sure that the ECAM status area still functions


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
